packages/vuetify/src/components/VBtn/VBtn.tsx

```diff
@@ -51,37 +49,30 @@ export default defineComponent({
     ...makeSizeProps(),
     ...makeTagProps({ tag: 'button' }),
     ...makeThemeProps(),
+    ...makeVariantProps({ variant: 'contained' } as const),
   }),
 
   setup (props, { slots }) {
     const { themeClasses } = useTheme(props)
     const { borderClasses } = useBorder(props, 'v-btn')
+    const { colorClasses, colorStyles, variantClasses } = useVariant(props, 'v-btn')
     const { roundedClasses } = useRounded(props, 'v-btn')
     const { densityClasses } = useDensity(props, 'v-btn')
     const { dimensionStyles } = useDimension(props)
     const { elevationClasses } = useElevation(props)
     const { positionClasses, positionStyles } = usePosition(props, 'v-btn')
     const { sizeClasses } = useSize(props, 'v-btn')
 
-    const isContained = computed(() => {
-      return !(props.text || props.plain || props.outlined || props.border !== false)
```

Comment:
We still have text/flat/plain/outlined props?

Reply:
Stylistically nothing has changed. Instead of being handled by javascript it's done by css now.